src/main/kotlin/app/revanced/patches/youtube/layout/hide/seekbar/patch/HideSeekbarPatch.kt

```diff
@@ -17,14 +17,16 @@ import app.revanced.patches.shared.settings.preference.impl.StringResource
 import app.revanced.patches.shared.settings.preference.impl.SwitchPreference
 import app.revanced.patches.youtube.layout.hide.seekbar.annotations.HideSeekbarCompatibility
 import app.revanced.patches.youtube.layout.seekbar.bytecode.patch.SeekbarColorBytecodePatch
+import app.revanced.patches.youtube.layout.seekbar.resource.SeekbarPreferencesPatch
 import app.revanced.patches.youtube.misc.integrations.patch.IntegrationsPatch
 import app.revanced.patches.youtube.misc.settings.bytecode.patch.SettingsPatch
 
 @Patch
 @DependsOn([
     IntegrationsPatch::class,
     SettingsPatch::class,
-    SeekbarColorBytecodePatch::class // Used to hide the seekbar in the feed and watch history
+    SeekbarColorBytecodePatch::class, // Used to hide the seekbar in the feed and watch history
```

Comment:
If `hide-seekbar` depends on this patch, will the seek bar be colored red by default? If so, including `hide-seekbar` patch also causes the unintentional side effect of affecting the color of the seek bar when not desired.

Reply:
I have a hard time inferring this. How come the `theme` patch is responsible for enabling coloring? I thought, that `SeekbarColorBytecodePatch` injects hooks, that color the seekbar by default.

Reply:
> But then again, the only thing SeekbarColorBytecodePatch does is inject calls to change the seekbar color. So maybe it's name is correct.

Which method is the hook calling? If it injects calls, don't these calls cause the seek bar that is responsible to change the color? So if those methods set the color to red by default (which is iirc what the seek bar color patch does) doesn't that mean that the `SeekbarColorBytecodePatch` patch would change the color?

Reply:
> if the custom seekbar color settings controls are available

How is this determined?

Reply:
I think I understand, so basically, `SeekbarColorBytecodePatch` does not change behavior if included because, by default, it does not color the seek bar (completely red) as previously assumed but instead follows YouTube's default behavior of coloring only when the player is focused.

Reply:
Thanks for clarifying. When I added the patch, the default behavior of it was to change the seek bar color to red completely by default, so I assumed this was still the case and hence questioned if hide-seekbar would cause this unintended change.